apps/e2e/cypress/e2e/talentsearch/redirects.cy.js

```diff
@@ -27,4 +27,26 @@ describe("Redirects", () => {
     cy.wait("@gqlgetMeQuery");
     expectToBeOnProfile();
   });
+
+});
+
+describe("Verify search page redirect", () => {
+const oldUrl = "/en/talent/search";
+const newUrl = "http://localhost:8000/en/search";
+
+it("redirects /en/talent/search", () => {
+  cy.visit(oldUrl);
+  cy.url().should("eq", newUrl);
+});
+it(" v301 redirect", () => {
+  cy.request({
+    url: oldUrl,
+    followRedirect: false, // turn off following redirects
+  }).then((resp) => {
+    // redirect status code is 301
+    expect(resp.status).to.eq(301);
+    expect(resp.redirectedToUrl).to.eq(newUrl);
+  });
+});
+
```

Comment:
a little nitpicky, but I think it would be good to cover redirects in /fr a bit too, rather than just english in tests
the other redirect tests were just english so far

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f605e02562826b1c90a3225061d560960fcfb8cd/apps/e2e/cypress/e2e/talentsearch/redirects.cy.js#L18-L29